src/Microsoft.HttpRepl/OpenApi/SchemaDataGenerator.cs

```diff
@@ -3,50 +3,51 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq;
+using Microsoft.OpenApi.Models;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.HttpRepl.OpenApi
 {
     public static class SchemaDataGenerator
```

Comment:
Is it OK to leave this as public?

Reply:
I switched it to internal like the others I changed. There's probably more that could be internal, but I'll make another pass at that later.